index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+  <title>Static Web Quiz</title>
+</head>
+<body>
+
+
+<article class="hatCard">
+  <header>
+    <h1 class="cardTitle">Swag Hat</h1>
+
+  </header>
+  <section>
+    <div><img src="hat1.jpg"></div>
+
+    <div class="description">This is the hat of an uncouth millenial.  Gives +2 to Charisma, but people will have a hard time respecting you.  Wear at your own risk.</div>
+
+    <div class="av">Available</div>
+  </section>
+
+  <section>
+    <header>Specifications</header>
+    <div class="bold">Size: 12" x 10" x 8"</div>
+    <div class="bold">Weight: 2.4 ounces</div>
```

Comment:
Consider making these specifications and pricing lists an unordered list. You can consolidate your class as well to something like this:
```
<ul class="bold>
  <li>Size: 12" x 10" x 8"</li>
   <li>Weight: 2.4 ounces</li>
</ul>
```